config/jobs/kubernetes/sig-network/sig-network-gce.yaml

```diff
@@ -1015,3 +1015,70 @@ periodics:
     testgrid-alert-stale-results-hours: '24'
     testgrid-alert-email: kubernetes-sig-network-test-failures@googlegroups.com
     description: Uses kubetest to run e2e Conformance, SIG-Network or Network Policy tests against a cluster (ubuntu based and uses containerd) created with cluster/kube-up.sh
+
+
+- interval: 6h
+  labels:
+    preset-k8s-ssh: "true"
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-gce-kops-nftables
+  decorate: true
+  decoration_config:
+    timeout: 90m
+  extra_refs:
+  - org: kubernetes
+    repo: kops
+    base_ref: master
+    workdir: true
+    path_alias: k8s.io/kops
+  spec:
+    serviceAccountName: k8s-kops-test
+    containers:
+    - command:
+      - runner.sh
+      args:
+      - bash
+      - -c
+      - |
+        make test-e2e-install
+        kubetest2 kops \
+          -v 2 \
+          --up --down \
+          --cloud-provider=gce \
+          --create-args="--channel=alpha --gce-service-account=default --networking=kindnet --set=spec.kubeProxy.proxyMode=nftables" \
```

Comment:
@hakman please check this is the correct syntax

Reply:
looks correct